v5/schemas.go

```diff
@@ -563,14 +563,15 @@ func (m NotificationResponse) IsRef() bool {
 }
 
 type Party struct {
-	Id          string       `json:"id,omitempty"`
-	Contact     *Contact     `json:"contact,omitempty"`
-	CreatedAt   int          `json:"created_at,omitempty"`
-	Roles       []string     `json:"roles,omitempty"`
-	Transaction *Transaction `json:"transaction,omitempty"`
-	UpdatedAt   int          `json:"updated_at,omitempty"`
-	UserId      string       `json:"user_id,omitempty"`
-	Object      string       `json:"object,omitempty"`
+	Id               string       `json:"id,omitempty"`
+	Contact          *Contact     `json:"contact,omitempty"`
+	CreatedAt        int          `json:"created_at,omitempty"`
+	Roles            []string     `json:"roles,omitempty"`
+	Transaction      *Transaction `json:"transaction,omitempty"`
+	UpdatedAt        int          `json:"updated_at,omitempty"`
+	UserId           string       `json:"user_id,omitempty"`
+	Object           string       `json:"object,omitempty"`
+	ClientVisibility int          `json:"client_visibility,omitempty"`
```

Comment:
Double-checking that the enum gets serialized to an int for this SDK? Looking at e.g., the `EntityType` enum on the `Contact` struct, I see it as an enum in `protos/src/contacts.proto` in Glide and serialized to a string in this SDK -- https://github.com/retitle/go-sdk/blob/84d4a86df00d5f59023a2ed9c12e443a54b4c4c0/v5/schemas.go#L59.

Reply:
Hi Harry, so you mean put the ClientVisibility as string instead of int right?

Reply:
Updated to be string

Reply:
I think so, based on that other example. Once we merge https://github.com/UrbanCompass/compass-glide-devapp/pull/1525 we can look at a sample response from the external API and confirm.